tensorflow/python/framework/ops.py

```diff
@@ -1461,9 +1461,7 @@ def convert_to_tensor(value,
   # TODO(b/142518781): Fix all call-sites and remove redundant arg
   preferred_dtype = preferred_dtype or dtype_hint
   if isinstance(value, EagerTensor):
-    if ctx is None:
-      ctx = context.context()
-    if not ctx.executing_eagerly():
+    if not (ctx or context).executing_eagerly():
```

Comment:
Could you revert this part of the PR. I'm not fond of calling executing_eagerly on the object in one case and the module in the other case.

Reply:
Good point, changed it in the latest commit